tests/eips/eip4844/test_blob_txs.py

```diff
@@ -447,13 +449,19 @@ def test_invalid_tx_max_fee_per_data_gas(
     pre: Dict,
     env: Environment,
     blocks: List[Block],
+    block_intermediate: Block,
 ):
     """
     Reject blocks with invalid blob txs due to:
 
     - tx max_fee_per_data_gas is barely not enough
     - tx max_fee_per_data_gas is zero
     """
+    if parent_blobs is not None:
```

Comment:
`parent_blobs` needs to be directly added as a parameter to this function, otherwise we are referring to the fixture function directly, which will never be `None`.

Reply:
Nice catch added here: https://github.com/ethereum/execution-spec-tests/pull/145/commits/5f05c1016fd9b2bd76a8c197f9257457f12e75c9